main/config.c

```diff
@@ -362,13 +362,14 @@ struct ast_config_include *ast_include_new(struct ast_config *conf, const char *
 	else
 		inc->included_file = ast_strdup(included_file);
 
-	inc->exec = is_exec;
-	if (is_exec)
+	inc->include_type = include_type;
+
+	if (include_type > 0)
 		inc->exec_file = ast_strdup(exec_file);
 
 	if (!inc->include_location_file
 		|| !inc->included_file
-		|| (is_exec && !inc->exec_file)) {
+		|| ((include_type > 0) && !inc->exec_file)) {
```

Comment:
No need for the inner parentheses

Reply:
Yes, the inner parentheses are not needed but they do help to eliminate any possible confusion.